tests/integration/test_charm.py

```diff
@@ -36,4 +36,11 @@ async def test_charm_crash(self, ops_test: OpsTest):
         await self.test_livepatch_server(ops_test)
         await simulate_charm_redeploy(ops_test)
         await self.test_livepatch_server(ops_test)
-        return
+
+    async def test_scale_up_and_down(self, ops_test: OpsTest):
+        """Scale the livepatch machine charm up and then down."""
+        await scale(application_name=APP_NAME, ops_test=ops_test, count=2)
+        await self.test_livepatch_server(ops_test, unit=0)
+        await self.test_livepatch_server(ops_test, unit=1)
+        await scale(application_name=APP_NAME, ops_test=ops_test, count=1)
+        await self.test_livepatch_server(ops_test, unit=0)
```

Comment:
Any way we can assert that the server is responding to requests?

Reply:
We can check for a `/debug/status` or `info` endpoint. Will add.